sdk/core/core-http/lib/util/xml.ts

```diff
@@ -16,12 +79,8 @@ export function stringifyXML(obj: any, opts?: { rootName?: string }) {
 }
 
 export function parseXML(str: string): Promise<any> {
-  const xmlParser = new xml2js.Parser({
-    explicitArray: false,
-    explicitCharkey: false,
-    explicitRoot: false
-  });
-  return new Promise((resolve, reject) => {
+  const xmlParser = new xml2js.Parser(xml2jsSettingsForXmlOperations);
+  const result = new Promise((resolve, reject) => {
```

Comment:
The constant `result` is not needed. Please revert the change.

Reply:
updated

Reply:
I don't see this changed.